### PR TITLE
add fixedaddress resource

### PR DESCRIPTION
**Pull request by epssy**
The fixedaddress resource requires WAPI_VERSION 1.1.

### CODE REVIEW

File: lib/infoblox.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@
 end
 
 module Infoblox
-  WAPI_VERSION   = '1.0'
+  WAPI_VERSION   = '1.1'
```
Comment (unclebilly) [inline lib/infoblox.rb]: @epssy This will break the gem for people who are not able to use 1.1, no? We should make this configurable.

Comment (sandlerr) [inline lib/infoblox.rb]: Perhaps `ENV['WAPI_VERSION'] || '1.0'`?

In that case we would need a guard on objects that aren't in 1.0. Another way would be to configure it on a per-resource basis.

Comment (unclebilly) [inline lib/infoblox.rb]: Yes, I think that would be an acceptable approach (using the environment variable).  I don't have access to the 1.1 API to test this - though  I'll certainly accept a PR.

**Comment by sandlerr** [comment]
@unclebilly what do you think of this approach? Would you prefer some kind of exception if using a resource that needs a more recent API than the one in the environment variable?

**Comment by unclebilly** [comment]
Thanks!

**Comment by unclebilly** [comment]
infoblox-0.2.18 has been released and pushed to rubygems with this change.  Thanks!

**Comment by sandlerr** [comment]
Thanks @unclebilly :D

**Pull request merged**